new-lamassu-admin/src/pages/Customers/components/TransactionsList.js

```diff
@@ -148,11 +148,7 @@ const TransactionsList = ({ customer, data, loading }) => {
       <div className={classes.titleWrapper}>
         <div className={classes.titleAndButtonsContainer}>
           <H4>
-            {loading
-              ? 'Loading'
-              : hasData
-              ? 'All transactions from this customer'
-              : 'No transactions so far'}
+            {loading ? 'Loading' : hasData ? '' : 'No transactions so far'}
           </H4>
```

Comment:
Probably better to hide the h4 as a whole instead of rendering it empty when it has data.